.editorconfig

```diff
@@ -7,3 +7,251 @@ trim_trailing_whitespace = true
+#cpp_align_multiline_type_argument                                              ; No override for "Align multiline template arguments" (Unspecified)
+cpp_align_multiline_extends_list = true                                         ; Align multiline base classes in class base clause
+cpp_align_multiline_ctor_init = true                                            ; Align multiline member initializers in member initializer lists﻿
+cpp_outdent_commas = true                                                       ; Outdent commas﻿ placed on new line
+cpp_align_ternary = none                                                        ; Do not align multiline ?: operator with first line (since alignment is incorrect)
+cpp_indent_aligned_ternary = false                                              ; Do not indent aligned ?: operator﻿ (since indentation varies)
+#cpp_align_multiline_calls_chain                                                ; No override for "Align multiline chained method calls" (Unspecified)
+#cpp_outdent_dots                                                               ; No override for "Outdent '.' and '->' in chained method calls﻿ on new lines" (Unspecified)
+cpp_align_multiline_binary_expressions_chain = false                            ; Do not align multiline chained binary expressions
+cpp_int_align_fix_in_adjacent = true                                            ; Fix column alignment in adjacent lines﻿
+cpp_int_align_eq = true                                                         ; Align assignments with adjacent assignments
+cpp_int_align_declaration_names = false                                         ; Do not align declaration names with adjacent declaration names
+cpp_int_align_comments = true                                                   ; Align end-of-line comments with adjacent end-of-line comments
+
+# Spaces #
+cpp_space_before_comma = false                                                  ; Do not put space after ANY commas
+cpp_space_after_comma = true                                                    ; Put space before ALL commas
+cpp_space_before_ptr_in_data_member = true                                      ; Put space before ptr in declaration of variable﻿
+cpp_space_after_ptr_in_data_member = false                                      ; Do not put space after ptr in declaration of variable﻿
+cpp_space_before_ptr_in_data_members = true                                     ; Put space before ptr in declaration of multiple variables﻿
+cpp_space_after_ptr_in_data_members = false                                     ; Do not put space after ptr in declaration of multiple variables﻿
+cpp_space_before_ptr_in_method = false                                          ; Do not put space before ptr in return type of function﻿
+cpp_space_after_ptr_in_method = true                                            ; Put space after ptr in return type of function﻿
```

Comment:
I can't quickly find a single function with `const char* Function()` compared to about 100 cases of `const char *Function()`